src/modules/actions.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-param-reassign */
+
+import populate from './injectHtml';
+
+// Save and get list function
+const save = (taskList) => localStorage.setItem('taskListArray', JSON.stringify(taskList));
+const getList = () => JSON.parse(localStorage.getItem('taskListArray')) || [];
+
+// sort array function
+const sort = () => {
+  const arrayT = getList();
+  const list = document.querySelectorAll('li');
+  arrayT.forEach((element, index) => {
+    arrayT[index].index = index;
```

Comment:
- [ ] I would advise you to add `+ 1` to the index so that it would give it a number from 1 to the list as it starts from zero at the moment.
![image](https://user-images.githubusercontent.com/86200356/220902409-f2f48323-407b-4152-81e2-2ed52c9709c3.png)